Azure/communication-ui-library
# Typing indicator fix for input method editor (IME)

# What


- Fixed typing editor for Android (it was caused by Android keyboard always returned keyCode as 229 and it's a known issue for Android)
- Added check for Enter key in `isEnterKeyEventFromCompositionSession`. For more information see https://github.com/Azure/communication-ui-library/issues/1425
- Added composition events handling for RTE

Steps to check:
- For Android: just check that typing indicator is shown when an Android user types in plain text and RTE editor
- For other platforms:
1. add Japanese or Chinese languages to use [IME](https://developer.mozilla.org/en-US/docs/Glossary/Input_method_editor)
2. Check that typing indicator is shown when typing in plain text and RTE editor
3. Input some text
4. Check that the first Enter key press **doesn't** send a message and just "confirms" the text
5. Check that the second Enter key press **sends** the message
6. Check the sent message looks as it should

# Why



# How Tested

Sample app on Android, iOS and MacOS Safari and MacOS Edge

# Process & policy checklist



**Is this a breaking change?**
